### PR TITLE
Make URL attributes into hyperlinks in the summary tab

### PR DESCRIPTION
**Pull request by jzstern**
Makes it so that attributes in the summary tab of the inspector view that are URLs are rendered as hyperlinks

### CODE REVIEW

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/summary.tsx [hunk 1]
```diff
@@ -432,153 +435,179 @@ const AttributeComponent = ({
...
+                                      />
+                                    </Typography>
+                                  )
+                                }
+                                return displayHighlightedAttrInFull(
+                                  lazyResult.highlights[attr],
+                                  val,
+                                  index
+                                )
+                              } else if (isUrl(val)) {
```
Comment (vinamartin) [inline ui-frontend/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/summary.tsx]: ❓ I think this is fine as is, but should we do something special if it is a link and also highlighted? I'll defer to @lifenouveau here.

**Review by vinamartin — APPROVED** [review]

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/summary.tsx (hunk 1 above)
Comment (lifenouveau) [reply ui-frontend/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/summary.tsx]: Theoretically the background would change, right?

Comment (vinamartin) [reply ui-frontend/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/summary.tsx]: Taking a look at this logic again, it only shows it as a url if it isn't highlighted. 🤔

Comment (lifenouveau) [reply ui-frontend/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/summary.tsx]: I see, ideally we have both.

Comment (jzstern) [reply ui-frontend/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/summary.tsx]: Ahhh I didn't realize highlighted meant exactly what you'd think it means. Will refactor so it links it for highlighted attributes as well.

Comment (jzstern) [reply ui-frontend/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/summary.tsx]: There's an issue where the <a> tag overwrites the inner highlight formatting, so I'm highlighting the entire link

**Review by lifenouveau — APPROVED** [review]

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/summary.tsx [hunk 2]
```diff
@@ -432,153 +435,191 @@ const AttributeComponent = ({
   const { isNotWritable } = useCustomReadOnlyCheck()
   const dialogContext = useDialog()
 
+  const isUrl = (value: any) => {
+    if (value && typeof value === 'string') {
+      const protocol = value.toLowerCase().split('/')[0]
+      return protocol && (protocol === 'http:' || protocol === 'https:')
+    }
```
Comment (vinamartin) [inline ui-frontend/packages/catalog-ui-search/src/main/webapp/component/tabs/metacard/summary.tsx:439]: If value doesn't exist or isn't a string, `return false`

**Comment by vinamartin** [comment]
✅ Manually verified LGTM

**Pull request merged**